bin/server.py

```diff
@@ -0,0 +1,80 @@
+"""Main module cusdeb-helpik that stores handlers"""
+import re
+import sys
+import requests
+from aiohttp import web
+import aiohttp_cors
+try:
+    from bin import config
+except ImportError:
+    print('Failed import config.py')
+    sys.exit(1)
+
+PORT = config.PORT
+MEDIAWIKI_URL = config.MEDIAWIKI_URL
+CLIENT_URL = config.CLIENT_URL
+
+PARAMS = {
+    'action': 'parse',
+    'page': '',
+    'prop': 'text',
+    'origin': '*',
+    'format': 'json',
+    'formatversion': 2,
+}
+
+SYNOPSIS = {
+    'text': '',
+    'url': '',
+}
+
+
+class Handler:
+    """Class contains handlers and auxiliary methods for them"""
+    def __init__(self):
+        pass
+
+    async def get_content_mediawiki(self, request):
+        """Method process GET request, takes page from client and return json file"""
+        PARAMS['page'] = request.query['Page']
+        SYNOPSIS['text'] = await self.get_synopsis()
+        SYNOPSIS['text'] = self._content_parser(SYNOPSIS['text'])
+        SYNOPSIS['url'] = MEDIAWIKI_URL+PARAMS['page']
+        return web.json_response(SYNOPSIS)
+
+    @staticmethod
+    async def get_synopsis():
+        """Method that parses the page content from mediawiki"""
+        session = requests.Session()
+        content = session.get(url=MEDIAWIKI_URL+'api.php', params=PARAMS)
```

Comment:
Define `api.php` as `API_URL` and use f-string for concatenate it to `config.MEDIAWIKI_URL`.